### ISSUE TITLE
Search in Tags when adding expense or income

### ISSUE THREAD

**Issue by chrisjluc** [mvarnagiris/financius#330]
Simple search icon at top right beside the 'add' icon that expands to search that can do prefix search matching (search as you type) or use Levenshtein search. I want to contribute.

**Comment by arulrajnet** [comment]
Yes. This feature is really needed. I have more than 50 tags. Every-time scrolling is too paining.
